CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+# **CODE OF CONDUCT**
+
+
+**All participants of Frontendweekly are expected to abide by our Code of Conduct, both online and during in-person events that are hosted and/or associated with Frontendweekly.**
+
+## **The Pledge**
+
+In the interest of fostering an open and welcoming environment, we pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
Body size is covered un personal appearance.

Reply:
What do you suggest?

Reply:
You can remove `body size`. It's redundant since you've mentioned `physical appearance`, which is more encompassing.

Reply:
Okay. On it!